dazel.py

```diff
@@ -146,27 +150,51 @@ def from_config(cls):
 
     def send_command(self, args):
         term_size = shutil.get_terminal_size()
-        command = "%s exec -i -e COLUMNS=%s -e LINES=%s -e TERM=%s %s %s %s %s %s %s %s %s" % (
+
+        docker_exec_command = "%s exec -i -e COLUMNS=%s -e LINES=%s -e TERM=%s %s %s %s %s" % (
             self.docker_command,
             term_size.columns, term_size.lines,
             os.environ.get("TERM", ""),
             "-t" if sys.stdout.isatty() else "",
             "--privileged" if self.docker_run_privileged else "",
             ("--user=%s" % self.user
              if self.user else ""),
-            self.instance_name,
+            self.instance_name)
+
+        if not self.user:
+            output_args = ("--output_user_root=%s --output_base=%s" % (
+                TEMP_BAZEL_OUTPUT_USER_ROOT, self.bazel_output_base)
+            if self.command and self.bazel_output_base else "")
+        else:
+            output_args = ("--output_user_root=%s" % (self.bazel_user_output_root)
+            if self.command and self.bazel_user_output_root else "")
+
+        command = "%s %s %s %s %s" % (
+            docker_exec_command,
             self.command,
             ("--bazelrc=%s" % self.bazel_rc_file
              if self.bazel_rc_file and self.command else ""),
-            ("--output_user_root=%s --output_base=%s" % (
-                TEMP_BAZEL_OUTPUT_USER_ROOT, self.bazel_output_base)
-             if self.command and self.bazel_output_base and not self.user
-             else  "--output_user_root=%s" % self.bazel_user_output_root
-                   if self.command and self.bazel_user_output_root
-                   else ""),
+            output_args,
             '"%s"' % '" "'.join(args))
-        command = self._with_docker_machine(command)
-        return os.WEXITSTATUS(os.system(command))
+
+        rc = os.system(command)
```

Comment:
This doesn't wrap things with docker machine anymore. is that on purpose?

Reply:
My misstake. Added back.